Adds doc for enabling Whisker (#2005)
tigera/docs: calico/observability/enable-whisker.mdx

---
description: Enable the flow logs API and Calico Whisker
title: Enable flow logs
---

# Enable the flow logs API and Calico Whisker

To generate and view flow logs, you need to enable the flow logs API (also known as Goldmane) and Calico Whisker.
These components are installed by default in new installations of Calico Open Source 3.30 and later.
They are not enabled by default if you upgraded from an earlier version.

There are two components to enable:
* **Goldmane**.

>   This is what Calico uses to generate the flow logs.

Reviewer: ```suggestion
  This is the part of Calico which aggregates the flow logs and provides a GRPC API to connect other tools to.
```

I believe the dataplane generates the logs in the first place, Felix grabs them and forwards them to Goldmane.

Author: Thanks, I'll clarify that statement.

  Goldmane can be used independently of Whisker.

>   For example, if you use Calico Cloud Free for observability, you can enable Goldmane and disable Whisker.

Reviewer: Or if you wish to write your own tool to do something with the stream of flow logs.

* **Whisker**.

>   This is what Calico uses to generate the Whisker console, a web-based interface for viewing flow logs.

Reviewer: ```suggestion
  This is the Calico component which presents the flow logs in the Whisker Console, a web-based interface for viewing flow logs.
```
